pkg/ebpf/bytecode/runtime/all_helpers.go

```diff
@@ -0,0 +1,219 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package runtime
+
+// updated as of Linux v6.0 commit 4fe89d07dcc2804c8b562f6c7896a45643d34b2f
```

Comment:
suggestion - document how to extract those symbols, for future use (debugging and reproducing in case of an helper is missing, or to add new helpers)